src/Hydrators/IssueHydrator.php

```diff
@@ -27,4 +29,35 @@ public static function getIssues(PDO $db, $completedFilter): array|null
         $query->setFetchMode(PDO::FETCH_CLASS, Issue::class);
         return $query->fetchAll();
     }
+    public static function getIssue($db, $issue_id)
+    {
+        $issueQuery = $db->prepare("SELECT `issues`.`id`,`issues`.`title`,`issues`.`description` AS 'summary', DATE_FORMAT(`issues`.`date_created`, '%d/%m/%Y') AS 'date_created', `issues`.`reporter`,`issues`.`department`, `issues`.`completed`,`severities`.`name` AS 'severity'
```

Comment:
You don't need to alias the `description` as 'summary'.

Reply:
Changes made to IssueHydrator and IssueDetails entity to correct this comment.